tools/editorconfig/.editorconfig

```diff
@@ -10,7 +10,7 @@ trim_trailing_whitespace = true
 [*.{yml,twig,php}]
 indent_size = 4
 
-[.travis-ci.yml]
+[{package.json,.travis-ci.yml}]
```

Comment:
And BTW, another `.json` files should be two spaced too. `composer.json` is the **only** exception.